src/main/java/chess/view/OutputView.java

```diff
@@ -0,0 +1,69 @@
+package chess.view;
+
+import chess.domain.location.Column;
+import chess.domain.location.Location;
+import chess.domain.location.Row;
+import chess.domain.piece.Piece;
+import chess.domain.piece.PieceType;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Map;
+
+public class OutputView {
+    public void printGameStart() {
+        System.out.println("> 체스 게임을 시작합니다.");
+        System.out.println("> 게임 시작 : start");
+        System.out.println("> 게임 종료 : end");
+        System.out.println("> 게임 이동 : move source위치 target위치 - 예. move b2 b3");
+    }
+
+    public void printBoard(Map<Location, Piece> board) {
+        Arrays.stream(Row.values()).sorted(Comparator.reverseOrder())
+                .forEach(row -> {
+                    printBoardRow(row, board);
+                    System.out.println();
+                });
+    }
+
+    private void printBoardRow(Row row, Map<Location, Piece> board) {
+        Arrays.stream(Column.values())
+                .map(column -> new Location(column, row))
+                .map(location -> board.get(location))
+                .map(this::convertPieceToString)
+                .forEach(System.out::print);
```

Comment:
```suggestion
        Arrays.stream(Column.values())
            .forEach(column -> {
                Location location = new Location(column, row);
                Piece piece = board.get(location);
                System.out.println(convertPieceToString(piece));
            });
```
map 이 여러번 반복되면 코드를 읽는사람에게 피로감을 주는것 같아요!

Reply:
미처 생각하지 못했던 부분인데, 다시 코드를 보니 정말 변경 과정을 따라가면서 인지적 부담이 늘어나네요. 
해당 부분은 `Row`와 다르게 역순 정렬이 필요 없기에 `for`문으로 리팩토링을 하였습니다.